### ISSUE TITLE
README suggests this is just QuickCheck rebranded

### ISSUE THREAD

**Issue by andreasabel** [hedgehogqa/haskell-hedgehog#437]
Hi, I was passing by here and looked into your shop window (README).
As QuickCheck user, I did not find anything that I not already had, so I was not attracted into your shop.

Maybe the README could be improved to tell me what hedgehog can do that QuickCheck cannot.  (Classic _related work_ section.)

**Comment by kvanbere** [comment]
Yes, the headline says "automatic" generation of test cases, but the example shows that the linear range is defined ahead of time, which I could just do in any other test library. I want to understand what it means by automatic.

**Comment by jacobstanley** [comment]
Yeah fair criticism!

The readme is currently not assuming anything about the reader, but obviously many (all?) Haskellers are or have been QuickCheck users.
